app/controllers/pages_controller.rb

```diff
@@ -26,8 +27,10 @@ def contribute
   def home_data
     setup_stories
     respond_to do |format|
-      format.json do
-        render json: home_data_json
+      if @stories
+        format.json do
+          render json: home_data_json
```

Comment:
last comment about a one-liner I promise 😄
```suggestion
          render json: home_data_json if @stories
```